rfcs/0000-template.md

```diff
@@ -0,0 +1,66 @@
+---
+Start Date: DD-MM-YYYY
+RFC PR: (leave this empty if no PR yet)
+Blade Issue: (leave this empty if no issue yet)
+---
+
+# Title of the RFC <!-- omit in toc -->
+
+## Table Of Contents <!-- omit in toc -->
+- [Summary](#summary)
+- [Basic Example](#basic-example)
+- [Motivation](#motivation)
+- [Detailed Design](#detailed-design)
+- [Drawbacks/Constraints](#drawbacksconstraints)
+- [Alternatives](#alternatives)
+- [Adoption strategy](#adoption-strategy)
+- [How Do We Educate People?](#how-do-we-educate-people)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+# Summary
+Brief explanation about the proposal.
+
+# Basic Example
+Include a basic code example. Omit this section if it's not applicable.
+
+# Motivation
+Why are we doing this? What use cases does it support? What is the expected outcome?
+
+Please focus on explaining the motivation so that if this RFC is not accepted, the motivation could be used to develop alternative solutions. In other words, enumerate the constraints you are trying to solve without coupling them too closely to the solution you have in mind.
```

Comment:
```suggestion
Try to focus on explaining the motivation so that if this RFC is not accepted, the motivation could be used to develop alternative solutions. In other words, enumerate the constraints you are trying to solve without coupling them too closely to the solution you have in mind.
```

Reply:
"enumerate the constraints you are trying to solve without coupling them too closely to the solution you have in mind."

Maybe some easier words for this? Seems too technical